tools/appveyor_install.ps1

```diff
@@ -0,0 +1,96 @@
+# Sample script to install Miniconda under Windows
```

Comment:
Agreed.  I think we'd just be adding one new top-level directory (`integration`) under which all of this CI/build management stuff would live.  Probably should vendor our .deb and .rpm specs there, too someday.